docs/configure.md

```diff
@@ -28,24 +28,3 @@ metadata:
 spec:
   controllerName: "apisix.apache.org/apisix-ingress-controller"
 ```
-
-### Addresses
```

Comment:
I think this section should be removed. It was missed out from from https://github.com/api7/api7-ingress-controller/pull/96.